src/main/kotlin/app/revanced/patches/vsco/misc/pro/fingerprints/RevCatSubscriptionFingerprint.kt

```diff
@@ -0,0 +1,15 @@
+package app.revanced.patches.vsco.misc.pro.fingerprints
+
+import org.jf.dexlib2.AccessFlags
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+
+object RevCatSubscriptionFingerprint : MethodFingerprint(
```

Comment:
Probably the strings property is enough to locate the method, the rest might be removed.

Reply:
No, because it might match this method:  

<img width="1105" alt="image" src="https://github.com/revanced/revanced-patches/assets/96280/8c09ecea-fda3-4e02-8a7c-aca13e0fd1ee">